manage.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
+  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.python.analysis`.*

The issue can be fixed by applying the following patch:

```diff
--- a/manage.py
+++ b/manage.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-  
+
 import os
 import sys
 
```